frontend/src/ui/components/DomainInput/DomainInput.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import { Button, FormControl, InputGroup, Spinner } from 'react-bootstrap';
+import { WrappedMessage } from 'utils/intl';
+import messages from './displayMessages';
+import './styles.scss';
+
+interface Props {
+  domainName: string;
+  internalDomain: boolean;
+  loading: boolean;
+  error: string;
+  handleDomainChange: Function;
+  handleSubmitDomain: Function;
+}
+
+export const DomainInput: React.SFC<Props> = (props: Props) => {
+  const isInvalid = !!props.error;
+  return (
+    <div className="domain-input-container">
+      <div className="domain-label">
+        <WrappedMessage messages={messages} id="typeDomainNameBelow" />
+      </div>
+      <InputGroup>
+        <FormControl
+          value={props.domainName}
+          onChange={(event: any) => {
+            props.handleDomainChange(event.target.value);
+          }}
+          isInvalid={isInvalid}
+        />
+        {props.internalDomain === true && (
+          <InputGroup.Append>
+            <InputGroup.Text>.opencraft.hosting</InputGroup.Text>
```

Comment:
Could we move this to the configs instead of hardcoding it? It reduces the risk of typos.